src/LayerPlan.cpp

```diff
@@ -95,8 +95,8 @@ LayerPlan::LayerPlan(const SliceDataStorage& storage, LayerIndex layer_nr, coord
 , last_extruder_previous_layer(start_extruder)
 , last_planned_extruder(&Application::getInstance().current_slice->scene.extruders[start_extruder])
 , first_travel_destination_is_inside(false) // set properly when addTravel is called for the first time (otherwise not set properly)
-, comb_boundary_minimum(computeMinimumCombBoundary())
-, comb_boundary_preferred(computePreferredCombBoundary())
+, comb_boundary_minimum(computeCombBoundary(true))
+, comb_boundary_preferred(computeCombBoundary(false))
```

Comment:
Rename the function computeCombBoundary to computeCombBoundaryWithMinimumBoundary, or else the boolean input can be confusing.

```suggestion
, comb_boundary_minimum(computeCombBoundaryWithMinimumBoundary(true))
, comb_boundary_preferred(computeCombBoundaryWithMinimumBoundary(false))
```

Reply:
Come to think of it, we can keep the function name the same and just introduce an enum CombingBoundary with values MINIMUM and PREFERRED. It will be more readable.

Reply:
Like the idea of an enum. I will use that